javascript/testing/testing-1.md

```diff
@@ -5,6 +5,13 @@ There are many test-running systems available in JavaScript: [Mocha](https://moc
 
 This lesson is going to center around __Jest__. The biggest reasons for this decision are that one of the best resources we've found for explaining JavaScript testing uses it and they have fantastic documentation.  In the end, writing tests is less about the syntax and more about the TDD philosophy.  The most important issues are knowing __why__ we write tests and __what__ we test rather than how.
 
+### Learning Outcomes
+By the end of this lesson, you should be able to do the following:
+
+- Know the basics of TDD.
```

Comment:
The phrasing on this first LO seems a bit off with the section opening blurb you used, since "knowing" something isn't really something that you can *do* to me. I would sub "explain" or something else in its place.